addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/AlertTags.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.commonlib;
+
+/** A standard set of alert tags. */
+public class AlertTags {
```

Comment:
Maybe `CommonAlertTags` or `OWASPAlertTags`?

How about using some sort of json, yaml, xml for these so that they could be i18n'd? (Like the base vulns in the core)

Reply:
Fine with the CommonAlertTags name, I wasnt thinking this would just be for OWASP tags.
I dont think tags should be i18n'ed at least not the ones we add. I'm expecting these sort of tags to be primarily used by tools.

Reply:
ok